src/AspNetCoreModuleV2/CommonLib/config_utility.h

```diff
@@ -7,24 +7,49 @@
 #include "ahutil.h"
 #include "stringu.h"
 #include "exceptions.h"
+#include "atlbase.h"
 
 class ConfigUtility
 {
     #define CS_ASPNETCORE_HANDLER_SETTINGS                   L"handlerSettings"
     #define CS_ASPNETCORE_HANDLER_VERSION                    L"handlerVersion"
+    #define CS_ASPNETCORE_DEBUG_FILE                         L"ASPNETCORE_MODULE_DEBUG_FILE"
```

Comment:
camelCase

Reply:
?

Reply:
we don't usually name web.config keys `ASPNETCORE_MODULE_DEBUG_FILE`. `debugFile` would be more appropriate.